django_project/alerts/tasks.py

```diff
@@ -128,9 +134,12 @@ def process_alert(setting: AlertSetting, runner: AnalysisRunner):
             )
             if name is not None or value is not None:
```

Comment:
@zamuzakki I think this will cause false positives, if name is not None but the value is, the check_treshold would be called with None.

Reply:
Updated to `and`. So it will only be called when name and value is not None.